core/common/src/main/java/alluxio/util/io/FileUtils.java

```diff
@@ -86,6 +87,45 @@ public static void changeLocalFileToFullPermission(String filePath) throws IOExc
   }
 
   /**
+   * Gets local file's owner.
+   *
+   * @param filePath the file path
+   * @return the owner of the local file
+   * @throws IOException when fails to get the owner
+   */
+  public static String getLocalFileOwner(String filePath) throws IOException {
+    PosixFileAttributes attr =
+        Files.readAttributes(Paths.get(filePath), PosixFileAttributes.class);
+    return attr.owner().getName();
+  }
+
+  /**
+   * Gets local file's group.
+   *
+   * @param filePath the file path
+   * @return the group of the local file
+   * @throws IOException when fails to get the group
+   */
+  public static String getLocalFileGroup(String filePath) throws IOException {
+    PosixFileAttributes attr =
+        Files.readAttributes(Paths.get(filePath), PosixFileAttributes.class);
+    return attr.group().getName();
+  }
+
+  /**
+   * Gets local file's permission.
+   *
+   * @param filePath the file path
+   * @return the file permission in posix format, e.g. "rwxr--r--"
+   * @throws IOException when fails to get the permission
+   */
+  public static String getLocalFilePermission(String filePath) throws IOException {
```

Comment:
simply iterating over the contents of `Files.readAttributes(Paths.get(filePath), PosixFileAttributes.class);` and computing the correct `short`